src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -75,7 +75,7 @@ class EditCourseForm extends React.Component {
     const languageOptions = (courseRunOptions &&
       [{
         label: '----------', // Add a default null language for the dropdown
```

Comment:
Nit: We should decide on consensus of how many dashes to put. Level type below uses two, here we have like 8